posthog/models/event/query_event_list.py

```diff
@@ -1,47 +1,49 @@
 import json
```

Comment:
Non-blocking: Let's 100% move this under `posthog/queries/event_list.py` and inline `SELECT_EVENT_FIELDS_BY_TEAM_AND_CONDITIONS_FILTERS_PART` and other constants within this file.